test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -163,22 +163,6 @@ void UcxContext::progress()
     progress_failed_connections();
 }
 
-void UcxContext::dispatch_new_connection(UcxConnection *conn)
-{
-    // To be implemented in a subclass
-}
-
-void UcxContext::dispatch_io_message(UcxConnection* conn, const void *buffer,
-                                     size_t length)
-{
-    // To be implemented in a subclass
-}
-
-void UcxContext::dispatch_connection_error(UcxConnection* conn)
-{
-    // To be implemented in a subclass
-}
-
```

Comment:
really need to remove?

Reply:
- easier to refactor maintain and debug
- actually dead code

Reply:
ok